src/ml/neural_net/mps_layer_instance_norm_data_loader.h

```diff
@@ -59,7 +59,9 @@ API_AVAILABLE(macos(10.14))
                          device:(id<MTLDevice>)dev
                       cmd_queue:(id<MTLCommandQueue>) cmd_q;
 
-- (void) updateIndex:(NSUInteger)style;
+- (NSUInteger) styleIndex;
+- (void) setStyleIndex:(NSUInteger)styleIndex;
```

Comment:
FYI you don't need to declare these. The property declaration above subsumes this

Reply:
Thanks, didn't know that!